src/testFixtures/groovy/grails/plugin/geb/ContainerGebSpec.groovy

```diff
@@ -85,19 +89,9 @@ abstract class ContainerGebSpec extends Specification implements ManagedGebTest,
         WebDriver driver = new RemoteWebDriver(webDriverContainer.seleniumAddress, new ChromeOptions())
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(30))
         browser.driver = driver
-    }
-
-    void setup() {
-        if (notInitialized) {
-            initialize()
```

Comment:
I fail to see how `initialize()` is called when there is no `setup()` method.

Reply:
I add the call in the spock extension.  Eventually we should move all logic there, but that requires moving config from class methods to annotations